calculation.js

```diff
@@ -14,3 +19,80 @@ function calculateBMI() {
 
     document.getElementById('result').innerHTML = `Your BMI is ${bmi}`;
 }
+
+
+       
+function TDEE() {
+    var feet = document.getElementById('feet').value;
+    var pounds = document.getElementById('pounds').value;
+    var age = document.getElementById('age').value;
+    var updatedvalue = document.getElementById("activity").value;
+    var fitnessvalue = document.getElementById("fitness").value;
+    if (!feet || !pounds || !age || !updatedvalue ||!fitnessvalue) {
+        alert("Please complete all fields.");
+        return; 
+    }
+    
+    
+
+    var inches = document.getElementById('inches').value || 0; 
+    var totalInches = (+feet * 12) + +inches; 
+    var heightInCM = totalInches * 2.54; 
+    var weightInKilos = pounds * 0.453592;
+    var bmr = (10 * weightInKilos) + (6.25 * heightInCM) - (5 * age) + 5;
+
+    var sed = 1.2;
+    var light = 1.375;
+    var mod = 1.55;
+    var very = 1.725;
+    var sup = 1.9;
+
+    
+
+    
+    var tdee;
+    if (updatedvalue=="Sedentary" && fitnessvalue =="lose") {
+        var tdee = (bmr * sed) - 500;
```

Comment:
So here's an example of why it's useful to declare constant variables.

a) You use the number 500 a dozen times throughout this code. Changing it means having to change it everywhere it's used.
b) It's not clear what the 500 value represents to someone reading this code. It takes a second to understand that it represents the daily calories gain/loss.

Think of a descriptive term for the `500` value, then declare at the top of the file, and use that variable everywhere.